src/actions/clusterActions.js

```diff
@@ -128,9 +128,16 @@ function clustersLoadV4(token, scheme, dispatch, getState) {
             return { id: clusterId, ...mockedStatus };
           } else {
             return clustersApi
-              .getClusterStatus(scheme + ' ' + token, clusterId)
-              .then(clusterStatus => {
-                return { id: clusterId, ...clusterStatus };
+              .getClusterStatusWithHttpInfo(scheme + ' ' + token, clusterId)
```

Comment:
Why this works if you have changed the function name but I dont see in the commit the change in any function definition? 🤔

Reply:
getClusterStatusWithHttpInfo is also available to use from the `giantswarm-js-client`, which is a generated client.

It returns the raw response, as well as the json parsed object.

So in the case i need to get the raw response and parse it myself. For a reason that I don't understand yet.